com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/DefaultRegistry.cs

```diff
@@ -9,9 +9,16 @@
 
 namespace UnityEditor.ShaderGraph.Registry.Default
 {
+    public class PropertyContext : IContextDescriptor
```

Comment:
Sweet, so this is the context node that properties will live on
Will there be a separate context node for keywords? Or will they share the same context node?

Reply:
Some more of the Context Node implementation and extension methods needs to be refreshed, but yea-- that's the gist here.